src/js/profile/mobile/widget/Calendar.js

```diff
@@ -73,9 +81,18 @@
 					calendarElement = self.element.querySelector(".calendarView");
 
 				switchElement = self.element.querySelector(".calendarSwitch");
+				arrowElement = self.element.querySelector(".calendarArrowLeft");
+
+				if (Calendar.pastSelection === "true") {
```

Comment:
You can use the 'self.options.pastSelection'.

Reply:
Since the default is false, I will add "!" at the beginning.